src/Hint/Typecheck.hs

```diff
@@ -37,17 +37,30 @@ kindOf type_expr =
        -- kind of errors
        failOnParseError parseType type_expr
        --
-       kind <- mayFail $ runGhc1 typeKind type_expr
+       (_, kind) <- mayFail $ runGhc1 typeKind type_expr
        --
        kindToString (Compat.Kind kind)
 
+-- | Returns a string representation of the normalized type expression.
+-- This is what @:kind!@ GHCi command prints after @=@.
+normalizeType :: MonadInterpreter m => String -> m String
```

Comment:
What's the reasoning behind the name? If this mimics `ghci`'s `:kind`, I would assume the name would be `kindOf`, but we already use that for something else. This is a bit confusing.